extensions/json-language-features/package.json

```diff
@@ -13,7 +13,8 @@
   "icon": "icons/json.png",
   "activationEvents": [
     "onLanguage:json",
-    "onLanguage:jsonc"
+    "onLanguage:jsonc",
+    "onLanguage:snippets"
```

Comment:
Your `snippet` extension will have to activate the JSON extension. Find it in `extensions.all` and calling `activate`.
That's not pretty, but currently the only way.

Reply:
`snippets` is the new grammar that is now part of the builtin json extension https://github.com/microsoft/vscode/pull/185017
since the pull request, schema and intellisense was disabled.
I found `"extensionDependencies"` in `package.json` to be better than `activate`